src/components/RefinementList.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div :class="suit('')" v-if="state">
+    <slot
+      :items="items"
+      :refine="state.refine"
+      :searchable="searchable"
+      :search-for-items="state.searchForItems"
+      :show-more="showMore"
+      :toggle-show-more="state.toggleShowMore"
+      :is-showing-more="state.isShowingMore"
+    >
```

Comment:
There is a couple of more values provided to the render function. Why we don't forward them? Like `isFromSearch`, `canRefine`, ...